internal/config/state.go

```diff
@@ -137,6 +137,8 @@ type LibraryState struct {
 	// Whether including this library in a release.
 	// This field is ignored when writing to state.yaml.
 	ReleaseTriggered bool `yaml:"-" json:"release_triggered,omitempty"`
+	// Whether to create a GitHub release for this library.
+	SkipGitHubReleaseCreation bool `yaml:"skip_github_release_creation" json:"-"`
```

Comment:
Should this be in the config.yaml? vs state.yaml?

Reply:
It should be in the `state.yaml` according to the issue.